examples/multigpu/node_classification_sage.py

```diff
@@ -171,12 +171,14 @@ def train(
     use_uva,
 ):
     # Instantiate a neighbor sampler
-    sampler = NeighborSampler(
-        [10, 10, 10],
-        prefetch_node_feats=["feat"],
-        prefetch_labels=["label"],
-        fused=(args.mode != "benchmark"),
-    )
+    if args.mode == "benchmark":
+        sampler = NeighborSampler([10, 10, 10], fused=False)
```

Comment:
so no prefetch is applied here which results in mismatch with GB example? how much performance improvement could prefetch bring?

Reply:
add comment here for this WAR?

Reply:
On my local machine, prefetching actually brings downgrade to the performance. If it doesn't do prefetch here, DGL performance will be better (4 gpus: 12s->10s)